README.md

```diff
@@ -173,7 +171,20 @@ This will create a binary with version `dev` and docker image pushed to
 
 ## Release a new version
 
-To release a new version first bump the version:
+To create the docker image and generate the manifests, go to the actions page on Github and click on Run Workflow . Specify the github tag that you want to create.
+Make sure that the tag is prefixed with `v`
+
+The workflow does the following:
+
+Runs make bump-version with the specified tag
+Creates the ccm related manifests file as <tag>.yaml
+Commits the manifest file under releases/ directory in the repo
+Creates release and tags the new commit with the input tag specified when workflow is triggered
+Login with dockerhub credentials specified as secrets
+Builds the docker image digitalocean/digitalocean-cloud-controller-manager:<tag>
+Pushes digitalocean/digitalocean-cloud-controller-manager:<tag> to dockerhub
```

Comment:
nit: move that above _"Builds the docker image [...]"_ since we first push the tag and then build/push the container image.

Reply:
This has 
Creates release and tags the new commit with the input tag
and builds docker image digitalocean/digitalocean-cloud-controller-manager:<tag> to dockerhub in that order. So, I don't understand what you mean..